src/main/java/io/dropwizard/primer/auth/filter/PrimerAuthAnnotationFilter.java

```diff
@@ -64,12 +64,11 @@ public PrimerAuthAnnotationFilter(final PrimerConfigurationHolder configHolder,
     @Override
     @Metered(name = "authorize")
     public void filter(ContainerRequestContext requestContext) throws IOException {
```

Comment:
Intent is to always stamp headers as downstream needs few headers (Device ID)

Verify behaviour is unchanged. It's still behind the whitelist